.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you revert this file to the one on master for now? I think we need to ask our tutor if we can add this to our .gitignore as different projects have different configuration. Anyway your checkstyle got disabled again.